File: challenges/LC-Two Sum.py
nums = [11, 2, 15, 7]
target = 9
i = 0
length = len(nums)

for f_index in range(length-1):
    j = i + 1
    while(j<length):
        if target - nums[i] == nums[j]:
            print(i,j)
            break
        j+=1
    i+=1

File: challenges/LC-Two Sum.js
var twoSum = function(nums, target) {
    let i = 0;
    let j = 0;
    
    for(i; i < nums.length; i++){
        j = i+1;
        //let k = j;
        for(j; j < nums.length; j++){
            if(nums[i]+nums[j]==target){
                return [i,j];
            }
        }
    }
};